org.eclipse.xtext.common.types.eclipse.tests/src/org/eclipse/xtext/common/types/xtext/ui/GenerateTypesTestLanguages.mwe2

```diff
@@ -1,80 +1,83 @@
 /*******************************************************************************
- * Copyright (c) 2010, 2017 itemis AG (http://www.itemis.eu) and others.
+ * Copyright (c) 2010, 2019 itemis AG (http://www.itemis.eu) and others.
  * All rights reserved. This program and the accompanying materials
  * are made available under the terms of the Eclipse Public License v1.0
  * which accompanies this distribution, and is available at
  * http://www.eclipse.org/legal/epl-v10.html
  *******************************************************************************/
-module org.eclipse.xtext.common.types.xtext.ui.GenerateTypesTestLanguages
+module org.eclipse.xtext.common.types.xtext.ui.GenerateTypesTestLanguage
```

Comment:
Since it generates more than one language, I'd prefer the old name

Reply:
c&p error. fixed